test/unit-test/Gen2SetTests.cpp

```diff
@@ -80,6 +82,15 @@ struct TestSuiteFixture {
 };
 }  // namespace
 
+//Verifies a timestamp exists and is of type string. Copies it to the
```

Comment:
I read your long comment:
```
The older VssCommandProcessorTests.cpp does not include this and has been "manually fixed. Opinions? If there is a preference to also use the helper, I can change it
```
I think the preference is there, it would be nice to change it, otherwise it may be forgot. But an utils file is not necessary for now I think. More important thing for me is using std lib instead of boost.